### PR TITLE
Update customActions.svelte Transition Container Update

### LINKED ISSUE #127: Minor Bug - transition:slide choppy / laggy
Cards > Custom Actions > Button (shows detailed information)
Currently docs and demo are using using:
```
{#if condition}
<div class='padding'>
    some text
</div>
{/if}
```
Will cause the padding to immediately show, and cause transition to jagged. 
Then the text appears smoothly resizing the container. 

I found the fix to be by simply adding another container - transition will make this the basis of the growth ease-in-out (assuming).

This seems to be the fix: (Double check me on that)
```
<div transition:slide>
    <div class="padding">
        {description}
    </div>
</div>
```

### PR DESCRIPTION
**Pull request by dmitrioligy**
If using {#if} with child container that has padding, the padding/margin makes the transition jagged (padding instantly appears before transition can finish)

To correct this, {#if} should have a child container that does the transition. The grandchildren can have whatever padding/margin needed.

tl;dr
```
{#if}
    <div transition>
        <div paddings here>
            {content}
        </div>
    </div
{/if}
```
Edit:
Closes #127

### CODE REVIEW

**Comment by Florian-Schoenherr** [comment]
Thank you!
Closes #127
